packages/cli/src/index.ts

```diff
@@ -37,12 +37,14 @@ program
 program
   .command('createapp <name>')
   .description('Create a new project.')
+  .option('-G, --no-git', "Don't initialize a git repository")
+  .option('-I, --no-install', "Don't autoinstall packages using yarn or npm (uses first available)")
   .option('-m, --mongodb', 'Generate a new project using Mongoose/MongoDB instead of TypeORM/SQLite')
   .option('-y, --yaml', 'Generate a new project using YAML configuration instead of JSON')
   .action((name: string, options) => {
     createApp({
-      autoInstall: true,
-      initRepo: true,
+      autoInstall: options.install,
```

Comment:
Should've commented, but this will always work because [commander always provides a boolean](https://www.npmjs.com/package/commander#other-option-types-negatable-boolean-and-flagvalue), but I will add a boolean coersion for typing.